lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -1,36 +1,33 @@
 <%page args="section_data" expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
-<div>
+<div class="action-type-container">
   %if section_data['is_small_course']:
   ## Show the gradebook for small courses
     <h3 class="hd hd-3">${_("Student Gradebook")}</h3>
-      <p>
-	${_("Click here to view the gradebook for enrolled students. This feature is only visible to courses with a small number of total enrolled students.")}
-      </p>
+      <label for="gradebook-link">
+    	${_("Click here to view the gradebook for enrolled students. This feature is only visible to courses with a small number of total enrolled students.")}
```

Comment:
@nasthagiri Up to you if out of scope for this PR, but we could make this similar to the "Click this link to view the student's progress page" below.
Replace "here" with "this link", and change the period after "enrolled students" to a colon, and move the "Gradebook" link to follow the colon. Change the link text from "View Gradebook" to "Gradebook".